insolar/flow/dispatcher/dispatcher.go

```diff
@@ -46,13 +46,15 @@ type Dispatcher struct {
 	PulseAccessor insPulse.Accessor
 }
 
-func NewDispatcher(present flow.MakeHandle, future flow.MakeHandle, past flow.MakeHandle) *Dispatcher {
+func NewDispatcher(pulseAccessor insPulse.Accessor, present flow.MakeHandle, future flow.MakeHandle, past flow.MakeHandle) *Dispatcher {
```

Comment:
if pulseAccessor is not required, maybe don't put it in constructor? most of the calls in this PR sends nil

maybe put it to setter

but I'm not sure how p, err := d.PulseAccessor.Latest(ctx) works at all

Reply:
discused, fixed